altsrc/input_source_context.go

```diff
@@ -9,6 +9,8 @@ import (
 // InputSourceContext is an interface used to allow
 // other input sources to be implemented as needed.
 type InputSourceContext interface {
+	Source() string
```

Comment:
Can we add a comment here to make it more clear for implementers what this should return?